src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -441,7 +470,8 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
 
       studentDeadlines: this.studentDeadlines,
       instructorDeadlines: this.instructorDeadlines,
-    }).pipe(finalize(() => {
+    }, isNotifyDeadlines)
+    .pipe(finalize(() => {
```

Comment:
Small nit: shuold we do 
```suggestion
    }, isNotifyDeadlines
    ).pipe(finalize(() => {
```